generator/src/main/java/com/graphicsfuzz/generator/fuzzer/OpaqueExpressionGenerator.java

```diff
@@ -84,6 +85,134 @@ public OpaqueExpressionGenerator(IRandom generator, GenerationParams generationP
 
   }
 
+  @FunctionalInterface
+  public interface OpaqueZeroOneFactory {
```

Comment:
Can you please move this interface into its own file?

And can you move the static methods for creating the lists of factories into OpaqueExpressionGenerator, and make them be non-static, rather than having them be static methods of the interface?

See comment below for (one reason) why this is a good idea.  In general I try to avoid an interface knowing about its implementing classes.